docs/RefMan/REPLCommands.rst

```diff
@@ -0,0 +1,376 @@
+    true. (If no argument, find an assignment for all properties.)
+
+    By default, this will return one assignment, but that can be updated with
+    ``:set satNum=VAL``
+
+``:set-seed SEED``
+    Seed the random number generator for operations using randomness.
+
+    A seed takes the form of either a single integer or a 4-tuple of unsigned
+    64-bit integers. Examples of commands using randomness are ``:dumpTests``
+    and ``:check``.
+
+``:t EXPR``, ``:type EXPR``
+    Check the type of an expression.
+
+``:time EXPR``
+    Measure the time it takes to evaluate the given expression.
+
+    The expression will be evaluated many times to get accurate results. Note
+    that the first evaluation of a binding may take longer due to laziness, and
+    this may affect the reported time. If this is not desired then make sure to
+    evaluate the expression once first before running ``:time``. 
+
+    The amount of time to spend collecting measurements can be changed with the
+    ``timeMeasurementPeriod`` option. 
+
+    Reports the average wall clock time, CPU time, and cycles. (Cycles are in
+    unspecified units that may be CPU cycles.)
+
+    Binds the result to
+
+    .. code-block:: cryptol
+
+        it : { avgTime : Float64
+                , avgCpuTime : Float64
+                , avgCycles : Integer }
+
+``:version``
+    Displays the version of the Cryptol executable.
+
+``:w FILE EXPR``, ``:writeByteArray FILE EXPR``
+    Write data of type ``fin n => [n][8]`` to a file.
+
+:set Options
+------------
+
+``:set base``
+    **Default value:** ``16``
+
+    **Valid values:** ``2``, ``8``, ``10``, ``16``
+
+    The base to display words at
+
+``:set debug``
+    **Default value:** ``off``
+
+    **Valid values:** ``off``, ``on``, ``false``, ``true``
+
+    Enable debugging output
+
+``:set ascii``
+    **Default value:** ``off``
+
+    **Valid values:** ``off``, ``on``, ``false``, ``true``
+
+    Whether to display 7- or 8-bit words using ASCII notation
+
+``:set infLength``
+    **Default value:** ``5``
+    
+    **Valid values:** any positive integer
+
+    The number of elements to display for infinite sequences
+
+``:set tests``
+    **Default value:** ``100``
+    
+    **Valid values:** any positive integer
+
+    The number of random tests to try with ``:check``
+
+``:set satNum``
+    **Default value:** ``1``
+
+    **Valid values:** any positive integer, ``all``
+
+    The maximum number of ``:sat`` solutions to display ("all" for no limits)
+
+``:set prover``
+    **Default value:** ``z3``
+
+    **Valid values:** ``cvc4``, ``cvc5``, ``yices``, ``z3``, ``boolector``, 
+    ``mathsat``, ``abc``, ``offline``, ``any``, ``sbv-cvc4``, ``sbv-cvc5``, 
+    ``sbv-yices``, ``sbv-z3``, ``sbv-boolector``, ``sbv-mathsat``, ``sbv-abc``, 
+    ``sbv-offline``, ``sbv-any``, ``w4-cvc4``, ``w4-cvc5``, ``w4-yices``, 
+    ``w4-z3``, ``w4-boolector``, ``w4-abc``, ``w4-offline``, ``w4-any``
```

Comment:
This list should also include [`bitwuzla`, `sbv-bitwuzla`, and `w4-bitwuzla`](https://github.com/GaloisInc/cryptol/pull/1787).